content/200-orm/200-prisma-client/500-deployment/301-edge/450-deploy-to-cloudflare.mdx

```diff
@@ -348,7 +348,7 @@ The command will output the URL where you can access the deployed Worker.
 If you are using a PlanetScale database, you need to:
 
 - use the `@prisma/adapter-planetscale` database adapter (via the `driverAdapters` Preview feature)
-- manually remove the conflicting `cache` field ([learn more](<https://developers.cloudflare.com/workers/databases/native-integrations/planetscale/#:~:text=fetch%3A%20(,init)%3B>)):
+- manually remove the conflicting `cache` field:
```

Comment:
removed the broken link: https://developers.cloudflare.com/workers/databases/native-integrations/planetscale/#:~:text=fetch%3A%20(,init)%3B

<img width="616" alt="image" src="https://github.com/user-attachments/assets/a0e0d95d-a4ff-4b6e-bb5b-7655bb7127b8" />